docs/crictl.1

```diff
@@ -1,5 +1,6 @@
 .nh
 .TH Container Runtime Interface (CRI) CLI
+.PP
```

Comment:
You need `v2` of md2man (`github.com/cpuguy83/go-md2man/v2`) to generate the docs for CI. :)